scripts/build-icons.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-console */
+
+import { Cheerio, Element, load } from 'cheerio'
```

Comment:
We could try [jsdom](https://github.com/jsdom/jsdom) which should be actively maintained. I used cheerio because it is the one used by React icons, so we could be sure to output the same artefacts

I can experiment with some alternatives in the context of this PR or, if you want to speed things up, I can track the issue and take it in consideration on a later time

Reply:
I think it's ok to open an issue and consider it later. It isn't production code, so it's ok

Reply:
They recently started updating the project again and released v1 of the library. So I think we can safely stick with cheerio for the time being